src/Illuminate/Database/Console/Migrations/BaseCommand.php

```diff
@@ -9,9 +9,13 @@ class BaseCommand extends Command {
 	 *
 	 * @return string
 	 */
-	protected function getMigrationPath()
+	protected function getMigrationPath($path = NULL)
 	{
-		return $this->laravel->databasePath().'/migrations';
+		if (! is_null($path)) {
+          return $this->laravel->basePath(). '/'. $path;
```

Comment:
Inconsistent spacing... Please remove the spaces between `.` and the strings. Also the indentation is wrong and the opening braces of the if-else belong onto their own line.